server/seeder.js

```diff
@@ -177,33 +77,29 @@ db.connectAsync()
         movie_shot_url,
         movie_cover_url
       ) VALUES (
-        '${movieNamesArray[Math.floor(Math.random()*movieNamesArray.length) -1]}',
-        '${tagsArray[Math.floor(Math.random()*tagsArray.length) -1]}',
-        '${tagsArray[Math.floor(Math.random()*tagsArray.length) -1]}',
-        ${1930 + Math.floor(Math.random()* 90)},
-        '${mpaaRatings[Math.floor(Math.random()*mpaaRatings.length) -1]}',
-        ${Math.floor(Math.random()*200)},
-        ${Math.random()*5},
-        ${Math.floor(Math.random()*5000)},
-        ${Math.floor(Math.random()*100)},
+        '${movieNamesArray[Math.floor(Math.random() * movieNamesArray.length) - 1]}',
```

Comment:
You may want to consider implementing a helper function to generate random numbers.